src/docs/schemaview.md

```diff
@@ -9,10 +9,4 @@ https://linkml.io/linkml/developers/schemaview.html#schemaview
 There are also several notebooks that demonstrate SchemaView usage in the LinkML-runtime repository:
 https://github.com/linkml/linkml-runtime/tree/main/notebooks
 
-Finally, the Biolink Model Toolkit is a helper wrapper around SchemaView for use with the Biolink Model and it
```

Comment:
@sierra-moxon you're the expert here so I trust you. Can you just say a little more about removing these lines?